web/packages/teleterm/src/ui/DocumentTerminal/Terminal/ctrl.ts

```diff
@@ -49,6 +52,8 @@ export default class TtyTerminal {
     AppConfig,
     'terminal.rightClick' | 'terminal.copyOnSelect'
   >;
+  private customKeyEventHandlers: Set<(event: KeyboardEvent) => boolean> =
+    new Set();
```

Comment:
Nit: I think it's my code but actually a more elegant way of declaring variables is to let TS infer the type:
```suggestion
  private customKeyEventHandlers = new Set<(event: KeyboardEvent) => boolean>();
```